starknet/src/authenticators/stark_sig.cairo

```diff
@@ -1,7 +1,6 @@
 use starknet::ContractAddress;
 use sx::types::{Strategy, IndexedStrategy, Choice};
 
-/// See https://community.starknet.io/t/snip-off-chain-signatures-a-la-eip712/98029
```

Comment:
would've been nice to keep this link somewhere in the codebase maybe for auditors?

Reply:
its in the stark eip712 lib